src/scripts/main.js

```diff
@@ -1,3 +1,60 @@
 'use strict';
 
-// write your code here
+// import:
+const {
+  mobileSwipes,
+} = require('./mobile');
+
+const {
+  arrowsPlayHandler,
+  createItemInRandomEmptyField,
+} = require('./move');
+
+// vars:
+const gameScore = document.querySelector('.game-score');
+const gameField = document.querySelector('.game-field');
+const allCells = gameField.querySelectorAll('.field-cell');
+const startBtn = document.querySelector('.button');
+const highestScore = document.querySelector('.highest-score');
```

Comment:
Usually we have a separated file for the variables, where we export them ad import where we need them, but in this case it's not critical since we use querySelector